Lib/test/test_tools/test_pathfix.py

```diff
@@ -39,29 +39,58 @@ def test_pathfix(self):
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python',
-                ['-i', '/usr/bin/python3',]),
-            '#! /usr/bin/python3',
-        )
+                ['-i', '/usr/bin/python3']),
+            '#! /usr/bin/python3',)
```

Comment:
PEP 8 prefers:

```suggestion
            '#! /usr/bin/python3')
```

Same remark for changes below.